indexer/db/solana_block_db.py

```diff
@@ -177,20 +178,37 @@ async def start(self) -> None:
             slot_list=DbSqlParam("slot_list"),
         )
 
+        priority_fee_percentiles_sql = DbSql(
```

Comment:
Please rename to:
`select_cu_price_percentile_list_sql`

Reply:
done